### ISSUE TITLE
Is it possible for bch to get burnt in the  non slp wallet during burning of slp tokens?

### ISSUE THREAD

**Issue by sriharikapu** [Bitcoin-com/slp-sdk#66]
I just want to know if there is a chance for BCH to get burnt on the NON SLP Wallets. Like bitcoin ABC core or other unsupported wallets. If there is any resource on this do share it in the comments.

**Comment by christroutner** [comment]
Yes. Traditional wallets that are not token-aware can and will burn SLP
tokens. You should only send tokens to a token aware wallet like Electron
Cash SLP or Badger Wallet (badger.bitcoin.com)

**Comment by sriharikapu** [comment]
Yes! @christroutner!! I came to know about that from the group and documentation but my corcern here is. Is there a chance of BCH to get burnt in non slp wallets during the burn event..!?
